src/components/search/curated/remix/index.tsx

```diff
@@ -2,13 +2,36 @@ import React from 'react'
 import {NextSeo} from 'next-seo'
 import Image from 'next/image'
 import groq from 'groq'
+import {TopicPageCourse} from 'types'
 import {HorizontalResourceCard} from 'components/card/topic-page-horizontal-resource-card'
 
-const SearchRemix = ({topic}: any) => {
+type TopicPage = {
+  topic: {
+    title: string
+    description: string
+    ogImage?: string
+    image: string
+    courses: TopicPageCourses
+  }
+}
+
+enum ResourceOrder {
+  primary = 'primary',
+  second = 'second',
+}
+
+type TopicPageCourses = {
+  [ResourceOrder.primary]: TopicPageCourse
+  [ResourceOrder.second]: TopicPageCourse
+}
+
+const SearchRemix = ({topic}: TopicPage) => {
   const location = 'Remix Topic Page'
   const description = `Build your Developer Portfolio and climb the engineering career ladder with in-depth Remix resources.`
   const title = `In-Depth Remix Resources for ${new Date().getFullYear()}`
-
+  const ogImage = topic.ogImage
+    ? topic.ogImage
+    : 'https://res.cloudinary.com/dg3gyk0gu/image/upload/v1654105781/egghead-next-pages/remix/og-image-remix-page.png'
```

Comment:
uses the og image that I made as a default but looks for an `og-image` property set (in images array) on `Remix Landing Page` in sanity